projects/1.18/assets/lazierae2/lazierae2/lang/zh_cn.json

```
@@ -1,66 +1,121 @@
 {
-  "itemGroup.lazierae2": "Lazier AE2",
+  "itemGroup.lazierae2": "懒人AE2",
```

```suggestion
  "itemGroup.lazierae2": "懒人AE2 1.16+",
```
MC百科译法，我不是很赞同，或者不译
懒人AE2是 Lazy AE2 的译名，lazier ae2 不建议沿用